<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalendarClass</class>
 <widget class="QMainWindow" name="CalendarClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calendar</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LayoutDirection::LeftToRight</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="Calendar">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="monthControlerWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>571</width>
        <height>56</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>571</width>
        <height>56</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="monthYearLabel">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>90</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
         <property name="text">
          <string>2026 January</string>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prevMonthButton">
         <property name="text">
          <string>Previous Month</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextMonthButton">
         <property name="text">
          <string>Next Month</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="calendarContainer" native="true">
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/Calendar.qrc"/>
 </resources>
 <connections/>
</ui>
